masquerade_tool/templates/masquerade_tool/index.html

```diff
@@ -0,0 +1,43 @@
+{% extends 'masquerade_tool/base.html' %}
+{% load static %}
+
+{% block breadcrumb %}
+  <nav>
+    <h3>
+      <a href="{% url 'dashboard_account' %}">Admin Console</a>
+      <small><i class="fa fa-chevron-right"></i></small>
+      Masquerade Tool
+    </h3>
+  </nav>
+{% endblock %}
+
+{% block content %}
+  <div class="container-fluid">
+    <div class="row">
+      <div class="col-med-9">
+        <br><br>
+        <p class="lead"><strong>Click the button below to temporarily assume the Masquerade role.</strong></p>
+        <p class="lead">Once granted, you'll be able to use
+          Canvas'
+          "<a href="https://community.canvaslms.com/t5/Admin-Guide/How-do-I-act-as-another-user-in-an-account/ta-p/161"
+            target="_blank">Act As User</a>" function for {{ session_mins }} minutes.
+        </p>
+      <form class="form-inline" method="POST" action="{% url 'masquerade_tool:add_role' %}">{% csrf_token %}
+          <button type="submit" name="masquerade_req" id="masquerade" class="btn btn-primary"
+            data-loading-text="<i class='fa fa-spinner fa-spin'></i> Requesting...">
+            <i class="fa"></i> Request Masquerade Access</button>
+      </form>
+
+      </div>
+    </div>
+  </div>
+{% endblock %}
+
+{% block javascript %}
+  {{ block.super }}
+  <script>
+    $('#masquerade').on('click', function () {
```

Comment:
Is this actually doing anything?

Reply:
Yeah, it displays the spinner on submission, and disables the button

Reply:
Ah, this must be some bootstrap/jquery ✨ magic ✨ . I can't find any documentation for it at first glance (this `button('loading')` feature) but 👍 if it works! Thanks.